play/src/components/joingame/EnterPlayerName.tsx

```diff
@@ -21,24 +24,28 @@ const PaddedContainer = styled(Box)(({ theme }) => ({
 }));
 
 interface EnterPlayerNameProps {
-  playerFirstName: string;
-  playerLastName: string;
-  handlePlayerFirstNameChange: (newValue: string) => void;
-  handlePlayerLastNameChange: (newValue: string) => void;
+  firstNameValue: string;
+  setFirstNameValue: (newValue: string) => void;
+  lastNameValue: string;
+  setLastNameValue: (newValue: string) => void;
+  isMobileDevice: boolean;
   inputError: boolean;
 }
 
 export default function EnterPlayerName({
-  playerFirstName,
-  playerLastName,
-  handlePlayerFirstNameChange,
-  handlePlayerLastNameChange,
+  firstNameValue,
+  setFirstNameValue,
+  lastNameValue,
+  setLastNameValue,
+  isMobileDevice,
   inputError,
 }: EnterPlayerNameProps) {
   const theme = useTheme();
+
   return (
     <JoinGameBackgroundContainer>
-      <StackContainer spacing={5}>
+      
+      <StackContainer spacing={isMobileDevice ? 2 : 5}>
```

Comment:
A 'mobile device' here is anything under 700px. Our Figma design includes desktop and 'mobile' designs, and then the responsive design handles tablet conditions:

![image](https://user-images.githubusercontent.com/79417944/234661639-a05c2475-395f-4a91-9540-179a75dbbe51.png)

This is differentiated is based on our breakpoint settings (sm=700px). 
Currently, the media query handles whether or not the answers are displayed in two columns or one and if a avatar on the avatar select screen is scaled down a bit. 

Overall though, I think `isMobileDevice` is a pretty ambiguous label. I've renamed it to `isSmallDevice` because that's as explicitly accurate as possible (it is small with respect to our breakpoints). If you've got a better suggestion though, would definitely appreciate it!